scripts/cleanup-translations.ts

```diff
@@ -1,5 +1,5 @@
 import { Buffer } from 'node:buffer'
-import flatten from 'flat'
+import { flatten } from 'flat'
```

Comment:
We can import `unflatten` as well:
```suggestion
import { flatten, unflatten } from 'flat'
```

and replace L54
```diff
-const unflattened = flatten.unflatten(targetTranslations)
+const unflattened = unflatten(targetTranslations)
```

Reply:
yeah, running a lot of stuff on my local: `pnpm dedupe && nr typecheck && nr test:typecheck && nr test:ci`